stage1/init/init.go

```diff
@@ -624,8 +625,8 @@ func stage1() int {
 	}
 
 	// kvm flavor has a bit different logic in handling pid vs ppid, for details look into #2389
-	// it does not require existance of "ppid", but instead registers current pid (which
-	// will be reused by lkvm binary) as an pod process pid used in entering
+	// it does not require existence of "ppid", but instead registers current pid (which
+	// will be reused by lkvm binary) as an pod process pid used in entering.
```

Comment:
While you are at it, you can add the missing articles

// It does not require the existence of "ppid", but instead registers the current pid...

s/an pod/a pod